### PR TITLE
Profile picture

### PR DESCRIPTION
**Pull request by MertzAndreas**
## Description

Register flow with upload image and multistep form. Also renamed some folders to comply with naming conventions.

 ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ 
 This means that a lot of files have been changed without anything happening to them other than that.

upright-point

## Type of PR

- Feature
- Code style update

## Related Issues/Tickets

#225 

## Checklist

- [X] Completed the user story described by the issue (if applicable)
- [X] Code follows the code guidelines
- [X] Tests have been added/updated (if applicable)
- [X] Documentation has been updated (if applicable)

### CODE REVIEW

File: apis/profileAPI.ts [hunk 1]
```diff
@@ -35,3 +35,30 @@ export const changePasswordRequest = async (userId: string | null, data: ChangeP
...
+
+  const imageData = {
+    uri: imageUri,
+    type: "image/jpeg",
+    name: "profile.jpg",
+  };
+
+  const formData = new FormData();
+
+  //TypeScript doesn't this format, so we need to typecast it
```
Comment (Tbjerregaard) [inline apis/profileAPI.ts:56]: 🤣

**Review by Tbjerregaard — CHANGES_REQUESTED** [review]
overall sehr gut

File: components/ProgressSteps.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  activeStep: {
+    backgroundColor: "#4CAF50",
+  },
+  inactiveStep: {
+    backgroundColor: "#D3D3D3",
+  },
+  stepContainer: {
+    flex: 1,
+  },
+});
```
Comment (Tbjerregaard) [inline components/ProgressSteps.tsx:77]: sharedStyles?

**Review by Tbjerregaard — CHANGES_REQUESTED** [review]

**Review by Tbjerregaard — APPROVED** [review]

**Review by MaksimHamko — APPROVED** [review]
Good code styling

**Pull request merged**